### PR TITLE
Add menu

### PR DESCRIPTION
**Pull request by GuenoleK**
### Description

Preparing the menu example component to be ready for the showcase

### CODE REVIEW

**Comment by pierr** [comment]
@GuenoleK  a rebase is needed

File: src/components/menu/example/index.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+const {MenuLeft} = FocusComponents.components;
+const Popin = FocusComponents.application.popin.component;
+
+const MyMenu = React.createClass({
+    GoHome() {
```
Comment (Bernardstanislas) [inline src/components/menu/example/index.jsx]: use camelcase, `goHome`

File: src/components/menu/example/index.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        return (
+            <Popin ref='popin' size='small'>
+                <h2>
+                    News
+                </h2><br/>
+                <h4>
+                    Hello there, <br/>
+                I'm a Popin component.<br/>
+            I Hope our demos are helpful...
+        </h4>
```
Comment (Bernardstanislas) [inline src/components/menu/example/index.jsx]: Strange indentation here :)

**Comment by GuenoleK** [comment]
Thank you for the advice @pierr

**Pull request merged**